### DISCUSSION TITLE
Clubic RSS don't work

### DISCUSSION THREAD

**Discussion by yodatak** [FreshRSS/FreshRSS#5198] [Q&A] [answered]
**Describe the bug**

When i try to add this rss feed 

I got this error
  | A feed could not be found at `https://www.clubic.com/feed/news.rss`;  the status code is `200` and content-type is `application/xhtml+xml`  [https://www.clubic.com/feed/news.rss]
-- | --

**Comment by Alkarex** [comment, marked as answer]
This feed is served with a wrong media type: https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.clubic.com%2Ffeed%2Fnews.rss

Try with `https://www.clubic.com/feed/news.rss#force_feed`

**Reply by yodatak** [reply]
Thanks it work !!
